Fun.Build/Types.fs

```diff
@@ -32,6 +32,7 @@ type StageContext = {
     TimeoutForStep: TimeSpan voption
     WorkingDir: string voption
     EnvVars: Map<string, string>
+    AcceptableExitCodes: Set<int>
```

Comment:
Is Set<int> a mutable type? If it is, can we change it to some imutable one like list? I know we have to distinguish it but because all other fields are imutable, I think we should keep it the same.

Reply:
Sorry, I just verified the Set type, it is immutable. 👍